templates/security/notices.html

```diff
@@ -58,30 +100,38 @@ <h2>Latest notices</h2>
       {% endfor %}
     </div>
 
-    <div class="col-3">
-      <div class="p-card">
-        <h2 class="p-heading--four">Subscribe</h2>
-
-        <ul class="p-list u-no-margin--bottom">
-          <li class="p-list__item u-sv1">
-            <a href="#" style="display: flex; align-items: center;">
-              <i class="p-icon--rss" style="margin-right: 0.5rem; flex-shrink: 0;"></i>
-              RSS
-            </a>
-          </li>
-
-          <li class="p-list__item">
-            <a href="#" style="display: flex; align-items: center;">
-              <i class="p-icon--email" style="margin-right: 0.5rem; flex-shrink: 0;"></i>
-              Mailing list
-            </a>
-          </li>
-        </ul>
+    {% if not search %}
+      <div class="col-3">
+        <div class="p-card">
+          <h2 class="p-heading--four">Subscribe</h2>
+
+          <ul class="p-list u-no-margin--bottom">
+            <li class="p-list__item u-sv1">
+              <a href="#" style="display: flex; align-items: center;">
+                <i class="p-icon--rss" style="margin-right: 0.5rem; flex-shrink: 0;"></i>
+                RSS
+              </a>
+            </li>
+
+            <li class="p-list__item">
+              <a href="#" style="display: flex; align-items: center;">
+                <i class="p-icon--email" style="margin-right: 0.5rem; flex-shrink: 0;"></i>
+                Mailing list
+              </a>
+            </li>
+          </ul>
+        </div>
       </div>
-    </div>
+    {% endif %}
   </div>
 
-  {% if featured_notices %}
+  {% if search %}
+    {% with %}
```

Comment:
`{% with total_pages = 5,  current_page = 1 %}` should work!

Reply:
You drive a hard bargain